extensions-contrib/distinctcount/src/main/java/io/druid/query/aggregation/distinctcount/DistinctCountAggregatorFactory.java

```diff
@@ -137,7 +136,8 @@ public AggregatorFactory getCombiningFactory()
   @Override
   public List<AggregatorFactory> getRequiredColumns()
   {
-    return Arrays.<AggregatorFactory>asList(new DistinctCountAggregatorFactory(fieldName, fieldName, bitMapFactory));
+    return Collections.singletonList(
+        new DistinctCountAggregatorFactory(fieldName, fieldName, bitMapFactory));
```

Comment:
It's not formatted properly, the closing `);` should be on the next line (or the whole expression on a single line).

Reply:
Fixed.